overrides/default-settings.gschema.override

```diff
@@ -136,18 +135,10 @@ active=false
 [org.gnome.settings-daemon.plugins.color]
 night-light-temperature=4500
 
-[org.gnome.settings-daemon.plugins.cursor]
-# Workaround upstream gnome-settings-daemon bug (https://bugzilla.gnome.org/show_bug.cgi?id=694758) as tracked by elementary (https://bugs.launchpad.net/elementaryos/+bug/1248747)
-active=false
-
 [org.gnome.settings-daemon.plugins.media-keys]
 screensaver='<Super>l'
 terminal='<Super>t'
 
-[org.gnome.settings-daemon.plugins.power]
-ambient-enabled=false
-idle-dim=false
```

Comment:
This is intentional. Instead of dimming the display on idle, we turn the display off; it felt weird to have both settings here and there's no visual way to toggle this one. I would be open to a counter-argument and more input from @elementary/ux.

Reply:
I can't recall if the idle-dim is happening at a smaller interval and then the display is turned off.
(dimming away)

I know at least, in Pantheon, the transition appears abrupt.
I'd have to investigate the default behavior and how it works in g-s-d though.